formapp/app.py

```diff
@@ -2249,8 +2251,9 @@ def endpoint_profile():
                 "fighter": dbcmd.execute("SELECT * FROM scores WHERE username=:uname AND game='fighter' ORDER BY score DESC", {"uname": user['name']}).fetchone(),
                 "wordle": dbcmd.execute("SELECT * FROM scores WHERE username=:uname AND game='wordle' ORDER BY score DESC", {"uname": user['name']}).fetchone(),
             }
+            print(highScores)
```

Comment:
Remove all prints that are not a part of standard debug logging.

Reply:
I have removed the prints ! ! ! ! ! !